src/pages/CreateRecipe.js

```diff
@@ -111,9 +141,9 @@ const ButtonStyled = styled.button`
   color: white;
 `
 
-const StyledDiv = styled.div`
+const StyledButtonDiv = styled.div`
   display: grid;
-  gap: 15px;
+  gap: 20px;
   flex-direction: column;
   margin-top: 40px;
 `
```

Comment:
CreateRecipe is a big file, maybe put some parts into an own component (inputs and buttons?)